ports/nrf/boards/WRL13990/mpconfigboard.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Roland van Straten
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#define MICROPY_HW_BOARD_NAME       "SPARKFUN WRL13990"
+#define MICROPY_HW_MCU_NAME         "NRF52832"
+#define MICROPY_PY_SYS_PLATFORM     "nrf52-DK"
+
+#define MICROPY_PY_MACHINE_UART     (1)
+#define MICROPY_PY_MACHINE_HW_PWM   (1)
+#define MICROPY_PY_MACHINE_HW_SPI   (1)
+#define MICROPY_PY_MACHINE_TIMER    (1)
+#define MICROPY_PY_MACHINE_RTCOUNTER (1)
+#define MICROPY_PY_MACHINE_I2C      (1)
+#define MICROPY_PY_MACHINE_ADC      (1)
+#define MICROPY_PY_MACHINE_TEMP     (1)
+#define MICROPY_PY_RANDOM_HW_RNG    (1)
+
+
+#define MICROPY_HW_HAS_LED          (1)
+#define MICROPY_HW_LED_COUNT        (1)
+#define MICROPY_HW_LED_PULLUP       (0) 
+
+#define MICROPY_HW_LED1             (7) // LED1
+
+// UART config
+#define MICROPY_HW_UART1_HWFC       (0)
+#define MICROPY_HW_UART1_RX         (26)
+#define MICROPY_HW_UART1_TX         (27)
+//#define MICROPY_HW_UART1_CTS        (7)
+//#define MICROPY_HW_UART1_RTS        (5)
+
+
+// SPI0 config
+#define MICROPY_HW_SPI0_NAME        "SPI0"
+#define MICROPY_HW_SPI0_SCK         (25)
+#define MICROPY_HW_SPI0_MOSI        (23)
+#define MICROPY_HW_SPI0_MISO        (24)
+
+// SPI1 config
```

Comment:
The defines for SPI1 would not have any effect as the machine/spi.c will only render SPI0 compile time. Does the Sparkfun breakout boards have a implicit board definition of where SPI1 is expected to be located? Like Arduino board layout expects SPI to be on pins located at the position of 11-13 and often CS on 10. If so, it could be an option to also update the modules/machine/spi.c to interpret a second compile time SPI. 

If there is no expectation on where SPI1 should be on the board i would probably have dropped this from the board definition as this would vary quite a lot between use cases of the board and would also require to implement the necessary hooks in the machine/spi.c to make them used. You can always define the pins used for mosi, miso, sck and cs from the machine.SPI module.

Reply:
@glennrub - thanks for your remark. I will change it as indicated. There is no "assigned" SPI1 whatsoever so lets make it as simple as possible, but not simpler as a famous person once said :-).